README.md

```diff
@@ -19,17 +19,24 @@ import { I18n, I18nFlavor } from "https://deno.land/x/grammy_i18n/mod.ts";
 
 ## Example
 
-Example project structure:
+An example project structure, you can also seperate your translations into different files to make it easier to maintain large projects.
+Nested translations don't change how you use translation keys, so nothing should break if you decide to use them.
```

Comment:
```suggestion
Nested translations don't change how you use translation keys, so nothing breaks when you decide to use them.
```

Reply:
Maybe also a different phrasing? (Thinking out loud)
```suggestion
Nested translations don't change how you use translation keys, so they stay them regardless of their containing file.
You can move them without change between files.
```

Reply:
I was planning to remove this line since (to me at least) it seems too similar to the one on line 38.
Perhaps we can just replace the whole paragraph with something like this:

"An example featuring both nested (`locales/en/...`) and standard (`locales/it.ftl`) file structure variants. Nested translations allow you to seperate your keys into different files, making it easier to maintain larger projects while also letting you use the standard variant at the same time. Using a nested file structure alongside the standard variant won't break any existing translations."

Reply:
i'd like to but due to not much time currently I prefer to skip it for now. I have more time in two weeks but I would hope its resolved until then 😇

Reply:
Alright. 

@strbit your suggestion works for me. Also, the more important docs are found on the website anyway, so I wouldn't want to nitpick this. Feel free to change it and then tag me or request a review when you're ready to have this merged.

Reply:
I’ll resolve this as I've implemented this suggestion in commit 08b431f 👌